api/app/dependency.py

```diff
@@ -0,0 +1,50 @@
+""" Authentication module for validating access tokens generated by Keycloak
```

Comment:
is `dependency.py` where we envision writing all our dependent modules in the future? I ask because if this isn't the case, the filename seems pretty vague

Reply:
Yes, open to better names. I will update the top comment missed.

Reply:
@andrea-williams  Suggesions? I've seen a convention for fastapi just naming in depends.py - but that doesn't seem any clearer to me.

Reply:
depends.py just reminds me of adult diapers. I'm not sure if a filename change is required (unless people like `dependencies.py`?), probably just updating the file's documentation so that it isn't auth-specific should clear up confusion

Reply:
I don't have a strong opinion either way. I'll wait for a decision to update this.

Reply:
I say update the module comments and leave filename as is.